sdk/src/main/java/com/hellocustomer/sdk/evaluation/EvaluationButtonBuilder.kt

```diff
@@ -40,8 +45,12 @@ internal data class EvaluationButtonBuilder(
         }
         QuestionTypeDto.CES -> Array(7) { index ->
             val scoreValue = index.plus(1)
+            val text = if (useCustomLabels){
+                labels[scoreValue]!!
```

Comment:
getValue() does not need not-null assertion :)

Reply:
but I need not null here for text variable